dbt_project.yml

```diff
@@ -582,6 +589,16 @@ seeds:
         +column_types:
           master_address: string
           alternative_address: string
+    nxm:
+      +enabled: true
+      +schema: test_data
+      ethererum:
+        nxm_ethereum_trades_seed:
+          +column_types:
+            evt_tx_hash: string
+            block_date: date
+            #TODO Please fill in column name and its type for the seed
+
```

Comment:
Hi @sharkxff , please follow the pattern and fill in the data type for the columns in the seed file.